src/main/java/com/ufcg/es5/BackendComplexoEsportivoUFCG/exception/constants/reservation/ReservationExeceptionMessages.java

```diff
@@ -1,9 +1,12 @@
 package com.ufcg.es5.BackendComplexoEsportivoUFCG.exception.constants.reservation;
 
-public class ReservationExeceptionMessages {
+public class ReservationExeceptionMessages extends IllegalArgumentException{
+
+    public static final String RESERVATION_WITH_ID_NOT_FOUND = "Reservation with id '%s' not found.";
+    public static final String RESERVATION_NOT_BELONGS_TO_USER = "Reservation with id '%s' does not belong to user '%s'.";
+    public static final String RESERVATION_LIMIT_EXCEEDED_IN_COURT_BY_USER = "Reservation limit exceeded for court '%s' by user '%s'.";
+    public static final String RESERVATION_wITHOUT_WAIT_MINIMUM_TIME_BY_USER_IN_COURT = "Reservation without wait minimum time by user '%s' in court '%s'.";
```

Comment:
RESERVATION_wITHOUT_WAIT_MINIMUM_TIME_BY_USER_IN_COURT -para USER_WITH_EMAIL_ALREADY_BOOKED_WITH_MINIMUM_INTERVAL_IN_COURT_WITH_NAME. e a mensagem deveria ser "User with email '%s' already booked with minimum interval in court with name '%s'.";